### ISSUE TITLE
{Linked:Bug3694274} [Adaptive Cards > Designer] - Extra navigation is observed after 'Apply reaction' button while navigating using swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8600]
### Target Platforms

Android

### SDK Version

1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Platform: Android
OS Version: 13
Screen reader: Talkback
Microsoft Outlook Version 4.2322.0 (42322810) dogfood

**Pre-Requisite:**

1. Go to this designer, which pins the version to 1.0. https://amdesigner.azurewebsites.net/
2. Paste the JSON
3. Click Send to send yourself an email of the adaptive card
4. Open it on your mobile device on the Outlook app to test it.

**Repro Steps:**
1. Open the outlook application.
2. Open the received mail having the same UI.
3. Perform right swipe and navigate to 'Apply Reaction' button.
4. Perform right swipe and observe the issue.

**Actual Result:**
Extra navigation is observed while navigating to the message section' from the 'apply reaction button'. On first right swipe focus is going on the reading pane web view and on performing second time right swipe it is announcing all details of section is one go then on performing third time swipe gesture it is going on the user defined image. 

**Expected Result:**
After performing right swipe from the 'Apply reaction button' talkback focus should be land on the next interactive element.

**Note:**
This issue is repro throughout the application.

**User Experience:** 
It will impact the screen reader user if the screen is not getting scrolled as they will not able to see the elements on the screen,

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-ssachan%40microsoft.com%7c68d0243104da4caeb08708d9ba4d6c6b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637745663988426007%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=YMsEOknKWOki3Fi%2BatsKabxS%2BM1tfTrgnws52jzpI7M%3D&reserved=0).

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/76aa6856-412b-410b-891f-58cce66188e8

### Card JSON

```JSON
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.0",
    "body": [
        {
            "type": "ColumnSet",
            "id": "1ede2aba-61b9-faa0-9895-9ed0c26b2e6f",
            "columns": [
                {
                    "type": "Column",
                    "id": "e5756242-0963-37a2-7cb4-4397886d60bb",
                    "padding": "None",
                    "width": "stretch",
                    "items": [
                        {
                            "type": "TextBlock",
                            "id": "20f3833e-0435-5c87-fad1-b528e0046fb6",
                            "text": "Company Name",
                            "wrap": true
                        }
                    ],
                    "verticalContentAlignment": "Center"
                },
                {
                    "type": "Column",
                    "id": "74215a26-fa8b-e549-cced-7f99fd34a661",
                    "padding": "None",
                    "width": "auto",
                    "items": [
                        {
                            "type": "Image",
                            "id": "795047e2-e63e-6e14-07ba-5a3e13323dff",
                            "url": "https://amdesigner.azurewebsites.net/samples/assets/PlaceHolder_Person.png",
                            "size": "Small"
                        }
                    ],
                    "horizontalAlignment": "Right"
                }
            ],
            "padding": {
                "top": "Small",
                "bottom": "Small",
                "left": "Default",
                "right": "Small"
            },
            "style": "emphasis"
        },
        {
            "type": "Container",
            "id": "fbcee869-2754-287d-bb37-145a4ccd750b",
            "padding": "Default",
            "spacing": "None",
            "items": [
                {
                    "type": "TextBlock",
                    "id": "44906797-222f-9fe2-0b7a-e3ee21c6e380",
                    "text": "[#83] Calls are getting dropped unexpectedly",
                    "wrap": true,
                    "weight": "Bolder",
                    "size": "Large",
                    "style": "heading"
                },
                {
                    "type": "TextBlock",
                    "id": "f7abdf1a-3cce-2159-28ef-f2f362ec937e",
                    "text": "We have this ongoing issue for the last one week. Outgoing calls from our account sometime get dropped unexpectedly. We need a quick resolution!",
                    "wrap": true
                },
                {
                    "type": "ActionSet",
                    "id": "c8a5ba7e-7ec6-53ae-79da-0cfb952a527e",
                    "actions": [
                        {
                            "type": "Action.ShowCard",
                            "id": "c5884df0-9c4d-aef4-9bf2-0ac49d064f75",
                            "title": "Change status",
                            "card": {
                                "type": "AdaptiveCard",
                                "padding": "Default",
                                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
                                "body": [
                                    {
                                        "type": "Container",
                                        "id": "c50c0c98-4cc1-6ff6-04b8-889577aaec86",
                                        "padding": "None",
                                        "items": [
                                            {
                                                "type": "Input.ChoiceSet",
                                                "id": "ChangeStatus",
                                                "placeholder": "Select an option",
                                                "label": "Select an option to change status. Please select an item in the list",
                                                "choices": [
                                                    {
                                                        "title": "Open",
                                                        "value": "Open"
                                                    },
                                                    {
                                                        "title": "Close",
                                                        "value": "Close"
                                                    }
                                                ],
                                                "isRequired": true
                                            },
                                            {
                                                "type": "ActionSet",
                                                "id": "ff6a8953-50c0-bf33-9d01-2316bc1ff50f",
                                                "actions": [
                                                    {
                                                        "type": "Action.Http",
                                                        "id": "17cf96b1-6455-a959-f88c-e301106c9c01",
                                                        "title": "Submit",
                                                        "url": "https://www.microsoft.com",
                                                        "method": "POST",
                                                        "body": "{ChangeStatus:{{ChangeStatus.value}}}"
                                                    }
                                                ]
                                            }
                                        ]
                                    }
                                ]
                            },
                            "isPrimary": true,
                            "style": "positive"
                        },
                        {
                            "type": "Action.ShowCard",
                            "id": "6698711a-d6d4-e3a1-9f8b-1042852d0b25",
                            "title": "Set priority",
                            "card": {
                                "type": "AdaptiveCard",
                                "padding": "Default",
                                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
                                "body": [
                                    {
                                        "type": "Container",
                                        "id": "6ba7238e-025b-e060-cf41-b4732848967b",
                                        "padding": "None",
                                        "items": [
                                            {
                                                "type": "Input.ChoiceSet",
                                                "id": "Priority",
                                                "placeholder": "Select a Priority",
                                                "label": "Select a Priority for the task.",
                                                "choices": [
                                                    {
                                                        "title": "Low",
                                                        "value": "Low"
                                                    },
                                                    {
                                                        "title": "Medium",
                                                        "value": "Medium"
                                                    },
                                                    {
                                                        "title": "High",
                                                        "value": "High"
                                                    },
                                                    {
                                                        "title": "Urgent",
                                                        "value": "Urgent"
                                                    }
                                                ],
                                                "style": "expanded",
                                                "isRequired": true
                                            },
                                            {
                                                "type": "ActionSet",
                                                "id": "827ea28d-3d41-91c6-cf84-30b74244afe8",
                                                "actions": [
                                                    {
                                                        "type": "Action.Http",
                                                        "id": "d499a757-ee04-ee57-ef75-4e3f01c536d1",
                                                        "title": "Submit",
                                                        "method": "POST",
                                                        "url": "https://www.microsoft.com",
                                                        "body": "{Priority: {{Priority.value}}}"
                                                    }
                                                ]
                                            }
                                        ]
                                    }
                                ]
                            }
                        },
                        {
                            "type": "Action.ShowCard",
                            "id": "f8541cdd-a532-3b7f-fca8-713b74529057",
                            "title": "Choose an assignee",
                            "card": {
                                "type": "AdaptiveCard",
                                "padding": "Default",
                                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
                                "body": [
                                    {
                                        "type": "Container",
                                        "id": "d2a2f235-2cf2-90c1-dab4-4a28dbea74d8",
                                        "padding": "None",
                                        "items": [
                                            {
                                                "type": "Input.ChoiceSet",
                                                "id": "Assignee",
                                                "placeholder": "Choose an assignee",
                                                "label": "Choose an assignee for the task.",
                                                "choices": [
                                                    {
                                                        "title": "kat.larrson@contoso.com",
                                                        "value": "kat.larrson@contoso.com"
                                                    },
                                                    {
                                                        "title": "daisy.philip@contoso.com",
                                                        "value": "daisy.philip@contoso.com"
                                                    },
                                                    {
                                                        "title": "miguel.garcia@contoso.com",
                                                        "value": "miguel.garcia@contoso.com"
                                                    },
                                                    {
                                                        "title": "collin.ballinger@contoso.com",
                                                        "value": "collin.ballinger@contoso.com"
                                                    },
                                                    {
                                                        "title": "elvia.atkins@contoso.com",
                                                        "value": "elvia.atkins@contoso.com"
                                                    },
                                                    {
                                                        "title": "lydia.bauer@contoso.com",
                                                        "value": "lydia.bauer@contoso.com"
                                                    },
                                                    {
                                                        "title": "henry.brill@contoso.com",
                                                        "value": "henry.brill@contoso.com"
                                                    },
                                                    {
                                                        "title": "mona.kane@contoso.com",
                                                        "value": "mona.kane@contoso.com"
                                                    }
                                                ],
                                                "style": "expanded",
                                                "isRequired": true
                                            },
                                            {
                                                "type": "ActionSet",
                                                "actions": [
                                                    {
                                                        "type": "Action.Http",
                                                        "id": "124e2030-8e95-2a1f-11f5-0da852070b0a",
                                                        "title": "Submit",
                                                        "method": "POST",
                                                        "url": "https://www.microsoft.com",
                                                        "body": "{Assignee:{{Assignee.value}}}"
                                                    }
                                                ]
                                            }
                                        ]
                                    }
                                ]
                            }
                        }
                    ]
                }
            ]
        },
        {
            "type": "Container",
            "id": "77102c5d-fde2-e573-4ea5-66022d646d64",
            "padding": {
                "top": "Small",
                "bottom": "Small",
                "left": "Small",
                "right": "Default"
            },
            "spacing": "None",
            "separator": true,
            "items": [
                {
                    "type": "TextBlock",
                    "id": "42654e7e-cece-b419-867a-3e3ef4076870",
                    "text": "[View in portal](https://amdesigner.azurewebsites.net)",
                    "wrap": true,
                    "color": "Accent",
                    "horizontalAlignment": "Right"
                }
            ],
            "horizontalAlignment": "Right"
        }
    ],
    "padding": "None"
}
```

**Comment by vagpt** [comment]
This issue can be tracked using below URL -

https://o365exchange.visualstudio.com/O365%20Core/_workitems/edit/3694274
